onetl/connection/db_connection/kafka.py

```diff
@@ -0,0 +1,197 @@
+    package_spark_2_3_3: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.3.3"  # noqa: WPS114
+    package_spark_2_3_4: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.3.4"  # noqa: WPS114
+    package_spark_2_4_0: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.4.0"  # noqa: WPS114
+    package_spark_2_4_1: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.4.1"  # noqa: WPS114
+    package_spark_2_4_2: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.4.2"  # noqa: WPS114
+    package_spark_2_4_3: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.4.3"  # noqa: WPS114
+    package_spark_2_4_4: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.4.4"  # noqa: WPS114
+    package_spark_2_4_5: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.4.5"  # noqa: WPS114
+    package_spark_2_4_6: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.4.6"  # noqa: WPS114
+    package_spark_2_4_7: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.11:2.4.7"  # noqa: WPS114
+    package_spark_3_0_0: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.0.0"  # noqa: WPS114
+    package_spark_3_0_1: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.0.1"  # noqa: WPS114
+    package_spark_3_0_2: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.0.2"  # noqa: WPS114
+    package_spark_3_0_3: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.0.3"  # noqa: WPS114
+    package_spark_3_1_0: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.1.0"  # noqa: WPS114
+    package_spark_3_1_1: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.1.1"  # noqa: WPS114
+    package_spark_3_1_2: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.1.2"  # noqa: WPS114
+    package_spark_3_1_3: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.1.3"  # noqa: WPS114
+    package_spark_3_2_0: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.2.0"  # noqa: WPS114
+    package_spark_3_2_1: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.2.1"  # noqa: WPS114
+    package_spark_3_2_2: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.2.2"  # noqa: WPS114
+    package_spark_3_2_3: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.2.3"  # noqa: WPS114
+    package_spark_3_2_4: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.2.4"  # noqa: WPS114
+    package_spark_3_3_0: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.3.0"  # noqa: WPS114
+    package_spark_3_3_1: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.3.1"  # noqa: WPS114
+    package_spark_3_3_2: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.3.2"  # noqa: WPS114
+    package_spark_3_4_0: ClassVar[str] = "org.apache.spark:spark-sql-kafka-0-10_2.12:3.4.0"  # noqa: WPS114
```

Comment:
Давай попробуем другой подход, текущий выглядит не очень:

```suggestion
    @classmethod
    def get_packages_for(cls, spark_version: str, scala_version: str | None = None) -> list[str]:
      if not scala_version:
        scala_version = "2.11" if spark_version.startswith("2") else "2.12"
      return [f"org.apache.spark:spark-sql-kafka-0-10_{scala_version}:{spark_version}"]
```

И зачем указывать в списке Spark 2.1, если минимально поддерживаемый 2.3?